templates/cc/string.go

```diff
@@ -4,14 +4,68 @@ const strTpl = `
 	{{ $f := .Field }}{{ $r := .Rules }}
 	{{ template "const" . }}
 	{{ template "in" . }}
-	{{ if $r.Pattern }}
+	{{ if $r.Len }}
```

Comment:
Please add an `{{ unimplemented }}` block, comment this out (see the `{{/*` below) and add a TODO for me. We don't currently generate UTF-8 length constraints in C++ and this definitely won't compile since it's Go code :)

Reply:
lol